.github/workflows/lint.yml

```diff
@@ -16,5 +16,5 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
-          version: v1.33
-          args: --timeout=5m
\ No newline at end of file
+          version: v1.45.2
```

Comment:
Does golangci-lint as an action mean we don't need to run golangci-lint in the build?  (#481 relates)

I'm about to throw in a PR to disable bench-race and golangci-lint for arm64, but if I can remove golangci-lint from both, that's even better.

Reply:
Yeah, it means it can be canned.